src/main/sql/admin_group.sql

```diff
@@ -0,0 +1,15 @@
+-- resource server: example group 'admin' for the addon-administration
+
+INSERT INTO scim_meta (id, created, lastmodified, location, resourcetype, version)
+VALUES (2, '2011-10-10', '2011-10-10', NULL, 'Group', NULL);
```

Comment:
we should make use of the [postgres](http://www.postgresql.org/docs/9.3/static/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT) and [mysql](https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_current-date) CURRENT_DATE function